pkgs/development/python-modules/pystac/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, buildPythonPackage
+, pythonOlder
+, setuptools
+, python-dateutil
+, html5lib
+, jsonschema
+, pytestCheckHook
+, pytest-cov
+, pytest-mock
+, pytest-recording
+, requests-mock
+}:
+
+buildPythonPackage rec {
+  pname = "pystac";
+  version = "1.10.1";
+  pyproject = true;
+  disabled = pythonOlder "3.9";
+
+  src = fetchFromGitHub {
+    owner = "stac-utils";
+    repo = "pystac";
+    rev = "v${version}";
```

Comment:
```suggestion
    rev = version;
```

Reply:
Why? prefix required here https://github.com/stac-utils/pystac/tags

Reply:
Sorry, I somehow overlooked it. Your are right.